### PR TITLE
Decouple join with HTTP endpoint from V3IO.

### PR DESCRIPTION
**Pull request by gtopper**

### CODE REVIEW

File: storey/flow.py [hunk 1]
```diff
@@ -278,21 +271,61 @@ async def _do(self, event):
...
+            else:
+                raise V3ioError(f'Failed to get item. Response status code was {response.status}: {response.body}')
+
+        JoinWithHttp.__init__(self, request_builder, join_from_response, **kwargs)
+
+    def _parse_response(self, response_body):
+        response_object = json.loads(response_body)["Item"]
+        for name, type_to_value in response_object.items():
+            val = None
+            for typ, value in type_to_value.items():
```
Comment (talIguaz) [inline storey/flow.py]: what about Timestamps and blobs?

File: tests/test_flow_integration.py [hunk 2]
```diff
@@ -1,21 +1,35 @@
...
+        client_session = aiohttp.ClientSession(connector=connector)
+        for i in range(1, 10):
+            request_body = json.dumps({'Item': {'secret': {'N': f'{10 - i}'}}})
+            response = await client_session.request(
+                'PUT', f'{self._webapi_url}/{table_path}/{i}', headers=self._put_item_headers, data=request_body, ssl=False)
+            assert response.status == 200, f'Bad response {response} to request {request_body}'
 
 
-@pytest.mark.skipif(not has_v3io_creds, reason='missing v3io credentials')
 def test_functional_flow():
```
Comment (talIguaz) [inline tests/test_flow_integration.py]: Add a test/example for the base `JoinWithHttp` as well

File: storey/test_v3io.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        'string': {'S': 'der die das'},
+        'boolean': {'BOOL': True},
+        'blob': {'B': base64.b64encode(b'message in a bottle').decode('ascii')},
+        'timestamp': {'TS': '1594289596:123456'}
+    }})
+    response = _v3io_parse_response(request)
+    expected = {
+        'int': 55,
+        'float': 55.4,
+        'string': 'der die das',
```
Comment (talIguaz) [inline storey/test_v3io.py:21]: זה ישראל פה

**Review by talIguaz — APPROVED** [review]

File: storey/test_v3io.py (hunk 3 above)
Comment (gtopper) [reply storey/test_v3io.py:21]: 🇮🇱

**Pull request merged**